osu.Game/Database/ArchiveModelManager.cs

```diff
@@ -404,7 +404,7 @@ private List<TFileModel> createFileInfos(ArchiveReader reader, FileStore files)
                 using (Stream s = reader.GetStream(file))
                     fileInfos.Add(new TFileModel
                     {
-                        Filename = FileSafety.PathSanitise(file),
+                        Filename = FileSafety.PathStandardise(file),
```

Comment:
I believe this stuff should be `FileSafety.PathStandardise(FileSafety.PathSanitise(file))` otherwise standardising `subdir\file.ext` will remain as is on unix systems.

On another note, this is not the first time `FileSafety` has caused us these issues.

Reply:
Should probably just change `PathStandardise` to fix this, to be honest..

Reply:
Prefer a framework fix.

Reply:
I merged the latest master so the fix is applied. I also removed a call to PathSanitise in LegacyStoryboardDecoder because it is now redundant. This should be resolved.